src/docfx/build/metadata/ContributionProvider.cs

```diff
@@ -99,7 +99,7 @@ async Task<Contributor> GetAuthor()
                     {
                         // Remove author from contributors if author name is specified
                         var (error, result) = await _gitHubUserCache.GetByLogin(authorName);
-                        errors.AddIfNotNull(error);
+                        errors.AddIfNotNull(error?.WithRange(authorName?.Range ?? default));
```

Comment:
can just be `authorName`

Reply:
this one takes a `Range` that are not implicitly convertible.

Reply:
I see